### ISSUE TITLE
Settings Error Layout: Settings > Data model Tables not showing properly in Mobile view.

### ISSUE THREAD

**Issue by ketanMehtaa** [twentyhq/twenty#7490]
## Bug Description

[Uploading Screencast from 08-10-24 05:15:04 PM IST.webm…]()

**Comment by ketanMehtaa** [comment]
@Bonapara , what will be the expected Behaviour , both the data and table header scrollable?

**Comment by ketanMehtaa** [comment]
In tableBody.tsx
`import styled from '@emotion/styled';
import { MOBILE_VIEWPORT } from 'twenty-ui';
const StyledTableBody = styled.div`
  display: flex;
  flex-direction: column;
  gap: 2px;
  padding: ${({ theme }) => theme.spacing(2)} 0;
  // added overflow
  @media (max-width: ${MOBILE_VIEWPORT}px) {
    overflow-x: auto;
  }
`;

export { StyledTableBody as TableBody };
`

**Comment by harshit078** [comment]
@ketanMehtaa, I think it should definitely not be scrollable and should be show all rows and columns under the table headers. Currently working on this area, will be pushing a PR by tomorrow !

**Comment by Bonapara** [comment]
In some situations, it will be hard to avoid horizontal scrolling, unfortunately, @harshit078, especially when there are four or more columns. In such cases, we might allow horizontal scrolling while keeping the row CTA sticky on the right edge.

But it's best to make it compact to stay on the screen without scrolling whenever possible.

**Comment by harshit078** [comment]
@Bonapara, Makes sense but shouldn't we only give scroll to smaller mobile viewports as large mobile viewports will handle it with ease. What is your take on this ?

**Comment by Bonapara** [comment]
Absolutely 💯

**Comment by charlesBochet** [comment]
Solved
